### PR TITLE
#2 - Make it Visual Studio 2019 compatible

### PR DESCRIPTION
**Pull request by Ruffo324**
See #2

### CODE REVIEW

**Comment by alexsorokoletov** [comment]
Thank you so much for your contribution! Give me some time to review and merge and release :)

File: SortAndRemoveOnSave/TextEditorExtension.cs [hunk 1]
```diff
@@ -20,8 +25,21 @@ protected override void Initialize()
 			DocumentContext.Saved += Document_Saved;
 		}
 
+
+
+
 		private void Document_Saved(object sender, EventArgs e)
 		{
+			var x = IdeApp.CommandService.GetCommands().ToList();
```
Comment (alexsorokoletov) [inline SortAndRemoveOnSave/TextEditorExtension.cs]: Do we need that? Looks like these lines were only for test?

File: SortAndRemoveOnSave/Properties/AddinInfo.cs [hunk 2]
```diff
@@ -1,14 +1,13 @@
...
+	"SortAndRemoveOnSave",
+	Namespace = "SortAndRemoveOnSave",
+	Version = "1.3"
 )]
 
 [assembly: AddinName("SortAndRemoveOnSave")]
 [assembly: AddinCategory("IDE extensions")]
 [assembly: AddinDescription("Visual Studio for Mac Add-in to sort and remove usings when saving C# file")]
-[assembly: AddinAuthor("Alex Sorokoletov")]
+[assembly: AddinAuthor("Alex Sorokoletov, Christian Groothoff(Fixes for VS 2019)")]
```
Comment (alexsorokoletov) [inline SortAndRemoveOnSave/Properties/AddinInfo.cs]: Christian, you can just add your name without “fixes for VS2019” if you’d like

**Review by alexsorokoletov — COMMENTED** [review]
.vs folder should be ignored and removed from that commit ideally. 
Would you have a chance to do that?

File: SortAndRemoveOnSave/TextEditorExtension.cs (hunk 1 above)
Comment (Ruffo324) [reply SortAndRemoveOnSave/TextEditorExtension.cs]: Only for test, i removed it.

File: SortAndRemoveOnSave/Properties/AddinInfo.cs (hunk 2 above)
Comment (Ruffo324) [reply SortAndRemoveOnSave/Properties/AddinInfo.cs]: Done

**Comment by Ruffo324** [comment]
Changed the few things.
If i got more time, i will refactore the code a bit more :)

Hope i successful removed the .vs directory. If not, then soon with a new Pull-Request :)

**Comment by Ruffo324** [comment]
@alexsorokoletov Aloha, is everything fine now?

**Review by alexsorokoletov — APPROVED** [review]

**Comment by alexsorokoletov** [comment]
Looks good to me! Thank you for the contribution and collaboration!

**Pull request merged**